README.rst

```diff
@@ -40,7 +40,7 @@ secure key to add. You should also have something like
        secure: "<your secure key from doctr here>"
 
    script:
-     - pip install requests cryptography sphinx
+     - pip install requests cryptography sphinx doctr
```

Comment:
Should also add 

    - pip install doctr

to the instructions in `__main__.py`.